Explorer/Assets/Scripts/ECS/StreamableLoading/Textures/LoadGlobalTextureSystem.cs

```diff
@@ -14,6 +15,15 @@ namespace ECS.StreamableLoading.Textures
     [LogCategory(ReportCategory.TEXTURES)]
     public partial class LoadGlobalTextureSystem : LoadTextureSystem
     {
-        internal LoadGlobalTextureSystem(World world, IStreamableCache<Texture2DData, GetTextureIntention> cache, IWebRequestController webRequestController) : base(world, cache, webRequestController) { }
+        internal LoadGlobalTextureSystem(World world, IStreamableCache<Texture2DData, GetTextureIntention> cache, IWebRequestController webRequestController, IGetTextureArgsFactory getTextureArgsFactory) : base(
+            world, cache, webRequestController, getTextureArgsFactory
+        ) { }
+
+        public override void Dispose()
+        {
+            base.Dispose();
+            //TODO move to the corresponding dispose point
+            getTextureArgsFactory.Dispose();
```

Comment:
So it will disappear as well, it's not a proper place anyway

Reply:
Where would you suggest to place the ownership of this obj?

Reply:
In the plugin/container